src/shim/kmq/bo.cpp

```diff
@@ -25,6 +27,14 @@ flag_to_type(uint64_t bo_flags)
   return AMDXDNA_BO_INVALID;
 }
 
+void flush_cache_line(const char *cur) {
```

Comment:
flush_cache_line() is also defined in umq/hwq.cpp. Maybe you want to move this to a header file in shim/ then this two files just include it?